packages/backend/src/studio.ts

```diff
@@ -66,6 +67,16 @@ export class Studio {
 
   public async activate(): Promise<void> {
     console.log('starting AI Lab extension');
+    this.telemetry = env.createTelemetryLogger();
+
+    // Ensure version is above 1.8.0
+    if (!version || lt(version, '1.8.0')) {
+      this.telemetry.logError('start.incompatible', {
+        version: version,
+        message: 'error activating extension on version bellow 1.8.0',
```

Comment:
there is a typo on bellow
```suggestion
        message: 'error activating extension on version below 1.8.0',
```